luxtronik/__init__.py

```diff
@@ -233,3 +266,49 @@ def _read_char(self):
         "Low-level helper to receive a signed int"
         reading = self._socket.recv(LUXTRONIK_SOCKET_READ_SIZE_CHAR)
         return struct.unpack(">b", reading)[0]
+
+
+class Luxtronik(LuxtronikData):
+    """
+    Wrapper around the data and the read/write interface.
+    Mainly to ensure backwards compatibility
+    of the read/write interface to other projects.
+    """
+
+    def __init__(self, host, port=LUXTRONIK_DEFAULT_PORT, safe=True):
+        super().__init__(Parameters(safe), Calculations(), Visibilities())
+        self.interface = LuxtronikSocketInterface(host, port)
+        self.read()
+
+    def read(self):
+        """
+        Read data from heat pump.
+        All available data will be read from the heat pump.
+        """
+        return self.interface.read(self)
+
+    def read_parameters(self):
+        """Read parameters from heat pump."""
+        return self.interface.read_parameters(self.parameters)
+
+    def read_calculations(self):
+        """Read calculations from heat pump."""
+        return self.interface.read_calculations(self.calculations)
+
+    def read_visibilities(self):
+        """Read visibilities from heat pump."""
+        return self.interface.read_visibilities(self.visibilites)
+
+    def write(self, parameters=None):
+        """
+        Write parameter to heat pump.
```

Comment:
I think that this doc string exists now twice?

Reply:
Once for `Luxtronik` and once for `LuxtronikSocketInterface`

Reply:
I think it would be sufficient to have it only once in the module?

Reply:
Done